utopian-io/v2.utopian.io
# [UT-418] - Auth i18n

### Verify that
---

This PR implements the internationalization module in the Auth app.

The implementation was based in the client app code.

### **Test plan**

* Run the app
* Go to `localhost:8080`
* If you're signed in, sign out and click in 'sign in' again
* Check that you were redirected to `localhost:8081/en/login`
* Check the string label of the sign in button
* Change the value of `auth.login.github` in the `i18n/locales_master/en.json` file and check that it updates the app
* Finish the sign in and check that you were redirected and are logged in the app
* Repeat the steps with German selected and check that the auth app is automatically set to German too.
